### ISSUE TITLE
OAuth 2.0 with Client Credentials not working

### ISSUE THREAD

**Issue by codedog** [usebruno/bruno#3021]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

I am trying to do OAuth with Client Credentials in Bruno, basically the equivalent of this Postman setup.

![Postman_OAuth2 0](https://github.com/user-attachments/assets/efb77693-d386-49c9-89c5-51f9f33a5391)

It works in Postman but I'm getting an error in Bruno. I can't see the actual request sent out by Bruno, and I'm not sure how I go about diagnosing this issue.

![Bruno_OAuth2 0](https://github.com/user-attachments/assets/b3360276-89dc-415f-98c5-b01d16f689ae)

### Screenshots/Live demo link

I have included screenshots in the description above.

**Comment by pietrygamat** [comment]
Client ID, secret and access token urls is a required field, you skipped it only for privacy, or you actually did not configure it?

Does it work in Postman if you select **client authentication** to send in Request body instead of Basic Auth header (possibly the cause, as brune uses the latter)?

**Comment by codedog** [comment]
> Client ID, secret and access token are required fields, you skipped it only for privacy, or you actually did not configure it?
> 
> Does it work in Postman if you select **client authentication** to send in Request body instead of Basic Auth header (possibly the cause, as brune uses the latter)? 

Yes, I configured those values but censored them in the screenshot.

The docs for this particular API mentions Basic Auth header, and it works. I haven't tried sending it in the body. What does the body content look like in Bruno's case? I can try it when I get back to the office.

**Comment by pietrygamat** [comment]
>  What does the body content look like in Bruno's case?
 It really is hard to explain in a few words, look at #1999. 

Also sending token in Basic Auth header in Bruno is not yet implemented: #2106

**Comment by codedog** [comment]
> Does it work in Postman if you select **client authentication** to send in Request body instead of Basic Auth header (possibly the cause, as Bruno uses the latter)?

I tried sending client credentials in the body in Postman, and it failed.

**Comment by pietrygamat** [comment]
That sounds about right, and it looks like you may have to wait for #2106 - [promised](https://github.com/usebruno/bruno/pull/2164#issuecomment-2324329296) to be delivered during this month. You may try running from the branch behind the link, and it hopefully works for you.

**Comment by anusree-bruno** [comment]
@codedog 

This must be working in v2. Could you configure and see?
https://docs.usebruno.com/auth/oauth2-experimental/collection-level-configuration

**Comment by codedog** [comment]
I manage my installation with Chocolatey. It seems the latest I can get there is 1.39.1.

**Comment by codedog** [comment]
> [@codedog](https://github.com/codedog)
> 
> This must be working in v2. Could you configure and see? https://docs.usebruno.com/auth/oauth2-experimental/collection-level-configuration

I manage my installation with Chocolatey. It seems the latest I can get there is 1.39.1.

**Comment by dsl3zm** [comment]
I would recommend using WinGet over chocolatey. You can pretty quickly install the latest version (currently 2.2.0) using `winget install --id=Bruno.Bruno  -e`

This is working with the latest version of bruno 

![Image](https://github.com/user-attachments/assets/7c51bd24-8d6e-4096-a076-8fbc77b3bbec)

**Comment by anusree-bruno** [comment]
We were having some issues with chocolatey packages see here https://github.com/usebruno/bruno/issues/4459 this is resolved now.

**Comment by anusree-bruno** [comment]
@codedog Hope you were able to install the latest version and get everything working. I'm closing this issue as resolved, but feel free to let us know if you’re still experiencing any problems.
